tests/__snapshots__/Choice.test.jsx.snap

```diff
@@ -20,7 +20,7 @@ exports[`test Renders Choice with an heading without crashing 1`] = `
         type="radio"
         value="opA" />
       <label
-        className=""
+        className=" "
```

Comment:
Empty spaces in the classes looks like something is broken. Trimming the class names will have as result more clear tests.